### DISCUSSION TITLE
ScriptReference/EditorGUILayout.TextArea

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#259] [Notes]
# ScriptReference/EditorGUILayout.TextArea

https://docs.unity3d.com/ScriptReference/EditorGUILayout.TextArea.html

**Comment by radiatoryang** [comment]
At least in 2018.2, you cannot normally access the TextEditor (to control keyboard cursor / caret position or selection) on EditorGUILayout.TextArea... unless you use reflection!   

```csharp
// see https://answers.unity.com/questions/411022/texteditor-in-editor-gui.html?childToView=1358721#answer-1358721  
TextEditor tEditor = typeof(EditorGUI).GetField("activeEditor", BindingFlags.Static | BindingFlags.NonPublic).GetValue(null) as TextEditor;
```
